hosts_example.yml

```diff
@@ -301,6 +301,26 @@ kafka_connect:
 #     syslog:
 #     elastic:
 
+#### Connectors and the Confluent Hub
```

Comment:
i meant to move this up below 
```
kafka_connect:
  vars:
```
There are other vars there, so just but them below